src/fidesops/api/v1/endpoints/health_endpoints.py

```diff
@@ -1,8 +1,7 @@
-from typing import Any, Dict
+from typing import Dict
 
 from fastapi import APIRouter
 
-
 router = APIRouter(tags=["Public"])
 
 
```

Comment:
if something like HEALTH= "/health" was added to urn_registry.py you could use that in the `router` decorator here as well as the test -